qa/suites/ceph-ansible/smoke/basic/2-ceph/ceph_ansible.yaml

```diff
@@ -4,6 +4,7 @@ meta:
 overrides:
    ceph_ansible:
      vars:
+        ansible-version: 2.4
```

Comment:
This entry isn't being used and the new ansible 2.5 default from https://github.com/ceph/teuthology/pull/1191 is being used instead.

Reply:
that's why we are seeing failures, @vasukulkarni ?

Reply:
I believe the issue is the ```ansible-version:``` needs to be moved up one level equal to ```vars:``` per this line: https://github.com/ceph/teuthology/blob/af590b821a72d8a60de794f1ae6fe700ae7a055b/teuthology/task/ceph_ansible.py#L28

Reply:
@Firefishy you mean like this?

 overrides:
    ceph_ansible:
      ansible-version: 2.4
      vars:

Reply:
Yes that is what I meant.

Reply:
@Firefishy looks like this is not correct